soh/src/overlays/misc/ovl_kaleido_scope/z_lmap_mark.c

```diff
@@ -37,8 +37,6 @@ extern PauseMapMarksData gPauseMapMarkDataTable[];
 extern PauseMapMarksData gPauseMapMarkDataTableMasterQuest[];
 
 void PauseMapMark_Init(PlayState* play) {
-    gBossMarkState = 0;
-    gBossMarkScale = 1.0f;
```

Comment:
any reason why you've removed this? I feel like this can cause issues with these values not being reset when the init is called again within the same game session.

Reply:
`PauseMapMark_Init` is actually called every time you call `PauseMapMark_Draw`. So what this does is reset the boss icon every frame, and it never pulsates.

The isn't really any harm in not resetting these values. All it means is that the icon may start small or large depending on what size it was when you closed the map. I don't think that's noticeable since it's changing size as the pause menu is folding up in it's opening animation, so you always first see it in the middle of a cycle anyways.